shop/src/main/java/ro/msg/learning/shop/controller/ProductController.java

```diff
@@ -0,0 +1,42 @@
+package ro.msg.learning.shop.controller;
+
+import org.springframework.web.bind.annotation.*;
+import ro.msg.learning.shop.model.Product;
+import ro.msg.learning.shop.service.IProductService;
+
+import java.util.List;
+
+@RestController
+public class ProductController {
+
+    private final IProductService productService;
+
+    ProductController(IProductService productService) {
+        this.productService = productService;
+    }
+
+    @GetMapping("/products")
```

Comment:
Here you should use the controller's `@RequestMapping` to have the prefix `products` in there rather than adding it to each mapping individually. In a previous controller you did this already